Localization/Config/en-US_Mods.ImproveGame.Configs.hjson

```diff
@@ -751,7 +751,7 @@ ImproveConfigs: {
 
 	PylonTeleNoBiome: {
 		Label: "[centeritem:TeleportationPylonMushroom] Biome-independent Pylon Teleportation"
-		Tooltip: "'Should I command such...'"
+		Tooltip: "'Your wish is my command'"
```

Comment:
After reconsidering the original line, I'd say "Because I said so!" fits the best.